omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/DelegatingCrudResource.java

```diff
@@ -91,7 +107,15 @@ public Object create(SimpleObject propertiesToCreate, RequestContext context) th
 		
 		T delegate = handler.newDelegate();
 		setConvertedProperties(delegate, propertiesToCreate, handler.getCreatableProperties(), true);
-		ValidateUtil.validate(delegate);
+		
+		try {

```

Comment:
This is not the thing that needs to be addressed based on the comments @lluismf added, in any case this is taken care of by BaseRestController.validationExceptionHandler.